umap/static/umap/js/modules/data/layer.js

```diff
@@ -1057,8 +1050,9 @@ export class DataLayer {
     const headers = this._reference_version
       ? { 'X-Datalayer-Reference': this._reference_version }
       : {}
-    await this._trySave(saveUrl, headers, formData)
+    const status = await this._trySave(saveUrl, headers, formData)
```

Comment:
`_trySave` doesn't always return a status, is that a problem?

Reply:
Here is how I think it works:
- when a layer conflicts, the whole save flow will stop (as we do not return a truthy status, maybe this should be more explicit?)
- user can click on the alert actions (override/cancel my edits), but AFAIK it's too late to proceed with the save flow (it's already stopped), we have to start it again

But I'll have a better look a it. I thought we cannot `await` for the user to click on the alert, but I did not test it for real, so I'll do!